ddtrace/propagation/http.py

```diff
@@ -197,11 +210,17 @@ def _inject(span_context, headers):
     @staticmethod
     def _extract(headers):
         # type: (Dict[str, str]) -> Optional[Context]
-        trace_id = _extract_header_value(
-            POSSIBLE_HTTP_HEADER_TRACE_IDS,
-            headers,
+        trace_id = int(
```

Comment:
the try/except from below is now removed for this.

we should be sure we have test cases that would cause this to fail (I am surprised no tests are failing).

Reply:
Do you mean this line: https://github.com/DataDog/dd-trace-py/blob/v1.10.1/ddtrace/propagation/http.py#L260?
`0 or None` evaluates to `None` so this behavior is unchanged.

Reply:
what if someone sends `X-Datadog-Trace-Id: peanuts` ?

Reply:
That would not be fun 😆

Reply:
I'll revert this change

Reply:
(well, or let's be sure we have a failing test for that to validate that my assumption is correct, or that we handle that case)